iOS/Views/ChannelHeaderView.swift

```diff
@@ -11,13 +13,35 @@ class ChannelHeaderView: UICollectionReusableView {
 
     @IBOutlet private weak var imageView: UIImageView!
     @IBOutlet private weak var descriptionLabel: UILabel!
+    @IBOutlet private weak var channelTeaserView: UIVisualEffectView!
+
+    weak var delegate: ChannelHeaderViewDelegate?
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        channelTeaserView.isUserInteractionEnabled = true
+        let tap = UITapGestureRecognizer(target: self, action: #selector(tappedPlayTeaserButton))
+        self.channelTeaserView.addGestureRecognizer(tap)
+    }
 
     func configure(for channel: Channel) {
+
         self.imageView.backgroundColor = channel.colorWithFallback(to: Brand.default.colors.window)
         self.imageView.sd_setImage(with: channel.imageURL, placeholderImage: nil)
         self.descriptionLabel.text = MarkdownHelper.string(for: channel.channelDescription ?? "").trimmingCharacters(in: .whitespacesAndNewlines)
+
+        self.channelTeaserView.isHidden = true
+
+        guard channel.stageStream != nil else { return }
+
+        self.channelTeaserView.isHidden = false
+        self.channelTeaserView.layer.roundCorners(for: .default)
```

Comment:
You can move this to `awakeFromNib` as this configuration is not based on the passed `channel` object.